packages/lesswrong/components/spotlights/SpotlightsPage.tsx

```diff
@@ -39,7 +44,7 @@ export const SpotlightsPage = ({classes}: {
     return [currentSpotlight, ...upcomingSpotlights, ...recycledSpotlights];
   }, [spotlights]);
 
-  const upcomingSpotlights = spotlightsInDisplayOrder.filter(spotlight => !spotlight.draft)
+  const [currentSpotlight, ...upcomingSpotlights] = spotlightsInDisplayOrder.filter(spotlight => !spotlight.draft)
```

Comment:
This isn't quite your fault, but I'm getting a bug here where. I have a spotlight with lastPromotedAt in the future. (Maybe that's not a supported use-case, and we have a case of [Hyrum's Law](https://www.hyrumslaw.com/).) It's making this code say that the "currentSpotlight" is that future one, whereas the one on the homepage of my dev forum is in the upcoming section. There's also a potential for position to have been recycled and dev data being a mess may be contributing. However I would note that nothing in this code is actually checking that the ""`currentSpotlight`"" is actually live.